### ISSUE TITLE
Installation error about generating the BVH

### ISSUE THREAD

**Issue by fjxiaopang** [Oslandia/building-server#5]
Hi,
I did the installation until  "./building-server-processdb.py conf/building.yml \<city\>" line. 
After that I get the error: " bash: syntax error near unexpected token `newline' "
Then, I add '\\' before '\\', that is, "./building-server-processdb.py conf/building.yml \\<city\\>".
After that I get another error: " ERROR: '\<city\>' city not defined in configuration file 'conf/building.yml' "
How can I solve the issue?
Any help would be greatly appreciated.

**Comment by Jeremy-Gaillard** [comment]
Hi,

You have to replace `<city>` by the name of the city defined in the building.yml configuration file that you wish to process. Make sure the configuration matches your local database parameters.

**Comment by fjxiaopang** [comment]
Thank you for your tips.
I use lyon (defined in building.yml ) to replace <city>, that is,
![qq 20170106141458](https://cloud.githubusercontent.com/assets/15902588/21709244/8a60ce56-d41a-11e6-8103-93c8dd68d5c7.png)
After that I get the error:
![qq 20170106141627](https://cloud.githubusercontent.com/assets/15902588/21709280/bc0a4a04-d41a-11e6-9355-665787fffc4f.png)
What should I do here?
@Jeremy-Gaillard

**Comment by Jeremy-Gaillard** [comment]
You have to set up a postgis database containing your geometries and modify the building.yml file to match your database profile.

You can find details on how to populate the database in [this workshop](https://github.com/Oslandia/workshop-3d-itowns/blob/master/0_Data/1_Buildings.md).

**Comment by Captain-Oski** [comment]
Hello everyone, my probleme in generating the BHV is that the Building-server-processDB.py can't find the table, even if my config file seems ok. 
![image](https://cloud.githubusercontent.com/assets/7432894/24509726/1fb5f0e8-1535-11e7-9b15-d39847696df5.png)
PostgreSQL logs, says : 
![image](https://cloud.githubusercontent.com/assets/7432894/24509767/47418064-1535-11e7-9265-cc1581d9d15b.png)

See my config file 
![image](https://cloud.githubusercontent.com/assets/7432894/24509823/6ced1508-1535-11e7-827a-11f0564b8e30.png)

See my postgreSQL architecture : 
![image](https://cloud.githubusercontent.com/assets/7432894/24509952/d038f92e-1535-11e7-8c62-3e45aaf318f2.png)

Any help would be welcome ! Thank you !

**Comment by Jeremy-Gaillard** [comment]
Are you sure the configuration file you linked is the same as the one saved on your disk? I managed to reproduce the issue by putting an empty field after "tablename:".

If that's not the cause of the issue, you could also check that no invisible characters are messing up the parsing.

If it still doesn't work, you could, as a temporary fix, modify the building-server-processdb.py file and directly assign `table` to `'itowns'` l34.

**Comment by Captain-Oski** [comment]
Thank you Jeremy for your quick answer. Unfortunatly, I tried a lot of different approaches, but with no different results... I decided to build a new Building-Server dist from scratch (new venv, setup.py OK, query the db ok) But when i try to initialize the BVH I run into the same error, which seems to come from psycopg2. I even tried to quote the sql statement (drop_column) , and what is strange is that the query time and quadtree creation time compute fine. then when it comes to add or drop column the psycopg2 lib do not find the db. I even hardcoded the db names and table names everywhere in the building-server-
processingdb.py 
![image](https://cloud.githubusercontent.com/assets/7432894/24557902/60aa4338-1607-11e7-9e25-eabab985eda6.png)
